src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -471,10 +468,9 @@ private void verifyTransitionOnPaidStatus(Invoice invoiceFromStorage, Invoice in
     // Once an invoice is Paid, it should no longer transition to other statuses, except Cancelled.
     if (invoiceFromStorage.getStatus() == Invoice.Status.PAID && invoice.getStatus() != Invoice.Status.CANCELLED &&
       invoice.getStatus() != invoiceFromStorage.getStatus()) {
-      List<Parameter> parameters = Collections.singletonList(new Parameter().withKey("invoiceId")
-        .withValue(invoice.getId()));
-      Error error = INVALID_INVOICE_TRANSITION_ON_PAID_STATUS.toError()
-        .withParameters(parameters);
+      var parameter = new Parameter().withKey("invoiceId").withValue(invoice.getId());
+      Error error = INVALID_INVOICE_TRANSITION_ON_PAID_STATUS.toError().withParameters(List.of(parameter));
+      logger.error("verifyTransitionOnPaidStatus:: Invalid invoice transition on paid status. '{}'", parameter);
```

Comment:
I don't see that toString() is overridden for Parameter class, so we will not have some meaningful messages.
Also this looks a bit overengineered, it would be enough just to add invoice Id to the log.
Also please change in another places as well

Reply:
Thanks, using only invoice getId